platform/tron/api.go

```diff
@@ -27,6 +27,32 @@ func (p *Platform) Coin() coin.Coin {
 	return coin.Coins[coin.TRX]
 }
 
+func (p *Platform) CurrentBlockNumber() (int64, error) {
+	return p.client.CurrentBlockNumber()
+}
+
+func (p *Platform) GetBlockByNumber(num int64) (*blockatlas.Block, error) {
```

Comment:
do we need pointer here?

Reply:
yes! to satisfy the platform methods inside the registry. And this pointer going to stay at a high level, don't have a chance to the garbage collector can't desalloc that